app/cdap/components/Connections/Create/reducer.ts

```diff
@@ -46,6 +46,10 @@ export interface IConnectorDetails {
   connectorDoc: unknown;
 }
 
+export interface IConnectorDetailsWithErrorMessage extends IConnectorDetails {
```

Comment:
I was not sure whether I should just add the new field into the original IConnectorDetails

Reply:
Is this used in all IconnectorDetails? if so feel free to add it? Unsure why you're not sure